File: javalin-openapi-processor/src/main/java/net/dzikoysk/openapi/processor/processing/AnnotationVisitor.java
package net.dzikoysk.openapi.processor.processing;

import javax.lang.model.element.AnnotationMirror;

final class AnnotationVisitor extends DefaultVisitor<AnnotationMirror, Void> {

    @Override
    public AnnotationMirror visitAnnotation(AnnotationMirror a, Void unused) {
        return a;
    }

}

File: openapi-processor/src/main/kotlin/com/dzikoysk/openapi/processor/processing/AnnotationVisitor.kt
package com.dzikoysk.openapi.processor.processing

import javax.lang.model.element.AnnotationMirror

internal class AnnotationVisitor : DefaultVisitor<AnnotationMirror, Void?>() {

    @Suppress("PARAMETER_NAME_CHANGED_ON_OVERRIDE")
    override fun visitAnnotation(annotationMirror: AnnotationMirror, unused: Void?): AnnotationMirror = annotationMirror

}